### PR TITLE
Range selector docs

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
Doc improvements to range selector 

#### Where should the reviewer start?
src/js/RangeSelector/doc.js
#### What are the relevant issues?
#2818 
#### Do the grommet docs need to be updated?
auto
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backward compatible

### CODE REVIEW

**Comment by britt6612** [comment]
Tests failed because Readme needs to be updated still in the process of checking them so just ignore for now

File: src/js/components/RangeSelector/doc.js [hunk 1]
```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
+    type: 'string',
+    defaultValue: 'accent-1',
+  },
+  'global.edgeSize': {
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: `global.edgeSize.small`

File: src/js/components/RangeSelector/doc.js [hunk 2]
```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
...
+    description: 'The possible sizes for the margin, padding and gap',
+    type: 'object',
+    defaultValue: 'small',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: `6px` I believe, but please double check. you had a similar prop on your Tabs PR, and you presented it the right way over there.

File: src/js/components/RangeSelector/rangeselector.stories.js [hunk 3]
```diff
@@ -56,9 +56,13 @@ class SimpleRangeSelector extends Component {
 
 const customThemeRangeSelector = deepMerge(grommet, {
   global: {
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/rangeselector.stories.js]: can you please add `borderSize` so we can see how it works?

File: src/js/components/RangeSelector/doc.js [hunk 4]
```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
...
+  },
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'string',
+    defaultValue: 'undefined',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: I think there should be a defaultValue for this one.  can you please double check?

File: src/js/components/RangeSelector/doc.js [hunk 5]
```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
...
+  },
+  'global.colors.border': {
+    description: 'The color for the border',
+    type: 'string',
+    defaultValue: 'undefined',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: for `global.colors` we usually do have defaultValues.
in this case it should be 
```
defaultValue: {
      dark: 'rgba(255, 255, 255, 0.33)',
      light: 'rgba(0, 0, 0, 0.33)',
    },
```
you can find it on `base.js`

**Comment by oorestisime** [comment]
@britt6612  thanks for jumping on this one!
But can we wait for https://github.com/grommet/grommet/issues/2802 ?
the theme values here are due to theme side effects. imho we should decide on a way and apply it everywhere.

File: src/js/components/RangeSelector/doc.js [hunk 6]
```diff
@@ -72,8 +72,37 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
-    description: 'The color foe the edge controls',
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: Can you please remove the double `T`

File: src/js/components/RangeSelector/doc.js [hunk 7]
```diff
@@ -72,8 +72,37 @@ export const doc = RangeSelector => {
...
+    type: 'string',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: should also support the color object

File: src/js/components/RangeSelector/doc.js [hunk 8]
```diff
@@ -72,8 +72,37 @@ export const doc = RangeSelector => {
...
+      xsmall: '1px',
+      small: '2px',
+      medium: '4px',
+      large: '12px',
+      xlarge: '24px,
+    }`,
+  },
+  'global.colors.border': {
+    description: 'The color for the border',
+    type: 'string',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: missing the object type

File: src/js/components/RangeSelector/doc.js [hunk 9]
```diff
@@ -72,8 +72,37 @@ export const doc = RangeSelector => {
...
+    defaultValue: {
+      dark: 'rgba(255, 255, 255, 0.33)',
+      light: 'rgba(0, 0, 0, 0.33)',
+    },
+  },
+  'global.colors.control': {
+    description: 'The color for the edge controls',
     type: 'string | { dark: undefined, light: undefined }',
```
Comment (ShimiSun) [inline src/js/components/RangeSelector/doc.js]: Can you please edit this line to `type: 'string | { dark: string, light: string }'`?

**Pull request merged**